test/uk/gov/hmrc/childcarecalculatorfrontend/views/ChildAgedThreeOrFourViewSpec.scala

```diff
@@ -33,7 +33,7 @@ class ChildAgedThreeOrFourViewSpec extends NewYesNoViewBehaviours {
 
   val location = Location.ENGLAND
 
-  def createView() = view(frontendAppConfig, BooleanForm(), NormalMode, location)(fakeRequest, messages)
+  def createView = () => view(frontendAppConfig, BooleanForm(), NormalMode, location)(fakeRequest, messages)
```

Comment:
Side note: do I understand this part correctly, that it could as well be a `val` now?